chart/values.yaml

```diff
@@ -75,14 +75,20 @@ appConfigValues:
   ## @param appConfigValues.ldsIsolineProvider The default LDS provider for isolines
   ldsIsolineProvider: ""
 
-## @section CARTO config parameters
-## Global configuration provided by CARTO. If you changed something from this section probably your self-hosted is going to stop working.
+## @section CARTO Replicated parameters
+## Global configuration for installations using Replicated. If you change something from this section probably your self-hosted is going to stop working.
 replicated:
   ## @param replicated.enabled Enable Carto replicated
   ## @param replicated.platformDistribution Platform distribution to deploy Carto Selfhosted with Replicated
+  ## @param replicated.ingressTestingMode Enable router testing mode for the application. This will just deploy the router in testing mode serving a simple page to check that it works.
+  ## @param replicated.onlyRunRouter Enable only the router component in the installation. This will just deploy the router component to check the ingress layer.
```

Comment:
```suggestion
  ## @param replicated.onlyRunRouter Enable only the router component in the installation. This will just deploy the router component. Useful to check the ingress layer together with the option `replicated.ingressTestingMode`.
```